common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -153,7 +153,7 @@ public void writeTo(ByteBuffer buffer) {
    *
    * Unlike getBytes this will not create a copy the array if this is a slice.
    */
-  public @Nonnull ByteBuffer getByteBuffer() {
+  @Nonnull public ByteBuffer getByteBuffer() {
```

Comment:
The compiler accepts it but I agree that annotations conventionally come before modifiers. They usually come on the line above.